Scheduler/observing_blocks_maintel/AOS/BLOCK-T75.json

```diff
@@ -0,0 +1,73 @@
+{
+    "name": "BLOCK-T75",
+    "program": "BLOCK-T75",
+    "reason": "tickets/SITCOM-1456",
```

Comment:
this also doesn't need to go here. Additionally, please change this to only SITCOM-XXX being that the ticket for the analysis